fuzz/fuzz_targets/flat_chainstore_header_insertion.rs

```diff
@@ -0,0 +1,98 @@
+#![no_main]
+
+use std::fs;
+use std::path::PathBuf;
+
+use bitcoin::block::Header as BlockHeader;
+use bitcoin::consensus::deserialize;
+use floresta_chain::pruned_utreexo::flat_chain_store::FlatChainStore;
+use floresta_chain::pruned_utreexo::flat_chain_store::FlatChainStoreConfig;
+use floresta_chain::pruned_utreexo::ChainStore;
+use floresta_chain::DiskBlockHeader;
+use libfuzzer_sys::fuzz_target;
+
+// Create a unique temporary directory for each fuzzing run
+fn create_temp_dir() -> PathBuf {
+    let rand_id = format!("fuzz-{}", std::process::id());
```

Comment:
Nah, it should use the buffer to create a random string to be the `rand_id`, I don't like this approach of using the process id.